gui/list.go

```diff
@@ -52,6 +54,7 @@ func UpdateMetadataLoop(m *model, delayToRefresh time.Duration) {
 
 func (m *model) Init() tea.Cmd {
 	m.playingItemIndex = m.list.Index()
+	m.volume = 100
```

Comment:
We should consider restoring previous volume on startup. Same goes mpv.Initialize() function, I forced the default volume to 100 but we should consider handling it properly if we're implementing the feature.

Reply:
Can be done in a separate MR 😬 🤡